vignettes/Metapopulation.Rmd

```diff
@@ -112,13 +111,45 @@ output <- aggregate(prev2to10 ~ mix + EIR + year, data = output, FUN = mean)
 Now let's visualize the results of mixing on PfPR2-10:
 
 ```{r}
-# plot
-ggplot(data = output) + 
-  geom_line(aes(x = year, y = prev2to10, color = factor(EIR))) +
-  facet_wrap(~ mix) +
-  scale_y_continuous(limits = c(0, 0.35)) + 
-  labs(x = 'time (years)',
-       y = 'PfPR 2-10 (month)',
-       color = 'EIR') +
-  theme_classic()
+
+colorBlindBlack8  <- c("#000000", "#E69F00", "#56B4E9", "#009E73", 
```

Comment:
Wow good for you for translating from ggplot ... 😃 
The figure is rendering a bit squished for me - can we make it like those in the treatment vignette where they fill the screen width wise?